src/components/Navbar/Navbar.jsx

```diff
@@ -13,10 +16,22 @@ const Navbar = () => {
       </div>
 
       <div className="nav-action">
-        <Link to="/" className="icon login" title="Login">
-          <i className="fa-solid fa-sign-in"></i>
-          <span>Login</span>
-        </Link>
+        {!isAuth ? (
+          <Link to="/login" className="icon login" title="Login">
+            <i className="fa-solid fa-sign-in"></i>
+            <span>Login</span>
+          </Link>
+        ) : (
+          <Link
+            to="/logout"
+            className="icon logout"
+            title="Logout"
+            onClick={() => setIsAuth(false)}
+          >
```

Comment:
As you are setting - token to localStorage while login, then when user logs out you can remove it from localStorage.

Reply:
Yes, I forgot to delete the token :') Thank you